test/mock_relay.cpp

```diff
@@ -664,34 +767,71 @@ TEST(relay, client_callback) {
   config.servers.push_back("fc02:2000::2");
   config.interface = "Vlan1000";
   config.state_db = state_db;
+  config.mux_table = mux_table;
   config.local_sock = -1;
 
+  std::unordered_map<std::string, struct relay_config> vlans;
+
+  uint8_t client_recv_buffer[] = {
```

Comment:
Is this variable used? It doesn't seem like it.

Reply:
It only used to get a valid dhcp client packet size. I leave it there for an example of good dhcp client packet, maybe it would help to understand the original function meaning in future.   :-)

Reply:
removed in latest diff.